### ISSUE TITLE
Form group with radio is causing all radio buttons having checked attribute

### ISSUE THREAD

**Issue by therajumandapati** [protonemedia/laravel-form-components#20]
We're using `<x-form-group>` with a couple of `<x-form-radio>` elements inside. When a value is selected and the form reloads due to validation errors, all radio elements have the `checked='checked'` attribute. 

![image](https://user-images.githubusercontent.com/5555261/95927774-556dc200-0d85-11eb-9f79-0db529b71979.png)

I think this is due to the fact that `<x-form-radio>` component is using a class which extends `FormCheckbox` and because of the presence of this line: https://github.com/protonemedia/laravel-form-components/blob/master/src/Components/FormCheckbox.php#L33

It could be fixed by changing `FormRadio.php` to the following: 

```
<?php

namespace ProtoneMedia\LaravelFormComponents\Components;

class FormRadio extends Component
{
    use HandlesValidationErrors;
    use HandlesBoundValues;

    public string $name;
    public string $label;
    public $value;
    public bool $checked = false;

    /**
     * Create a new component instance.
     *
     * @return void
     */
    public function __construct(
        string $name,
        string $label = '',
        $value = 1,
        $bind = null,
        bool $default = false,
        bool $showErrors = true
    ) {
        $this->name       = $name;
        $this->label      = $label;
        $this->value      = $value;
        $this->showErrors = $showErrors;

        if (old($name) == $value) { <== CHANGE
            $this->checked = true;
        }

        if (!session()->hasOldInput() && $this->isNotWired()) {
            $boundValue = $this->getBoundValue($bind, $name);

            $this->checked = is_null($boundValue) ? $default : $boundValue == $value; <== CHANGE
        }
    }
}
```

**Comment by pascalbaljet** [comment]
Thans for reporting, I'll try to fix it this week.

**Comment by InWave** [comment]
Thanks @therajumandapati, you saved me quite some time!

**Comment by pascalbaljet** [comment]
Could you both try this branch and confirm it's been fixed? Thanks!

```bash
composer require protonemedia/laravel-form-components:dev-radio-bugfix-20
```

**Comment by pascalbaljet** [comment]
@InWave @therajumandapati any news on testing the bugfix?

**Comment by InWave** [comment]
Will check now!

**Comment by InWave** [comment]
Just tested it and it works on my end. Thanks @pascalbaljet

**Comment by pascalbaljet** [comment]
Thanks, I'll merge it soon!

**Comment by pascalbaljet** [comment]
Fixed in [v2.1.2](https://github.com/protonemedia/laravel-form-components/releases/tag/2.1.2)

**Comment by therajumandapati** [comment]
Thanks for the fast turn around @pascalbaljet
